Cod3rsGrowth.Dominio/Colors.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Cod3rsGrowth.Dominio
+{
+    public enum Colors
+    {
+        Energy = 'E',
+        White = 'W',
+        Blue = 'U',
```

Comment:
Se for utilizar assim, os letras devem funcionar como uma descrição, altere para ficar nesse padrão 

![Screenshot_1](https://github.com/silvamarcospaulo/Cod3rsGrowth/assets/124618635/2c05916b-a6a5-42e1-bbf8-b62d214477a0)